scripts/ubuntu/install-compiler-prereqs.sh

```diff
@@ -65,8 +70,10 @@ TEXT_RESET=`tput -T xterm-256color sgr0`
 # Install misc dependencies
 echo "${TEXT_GREEN}Installing dependencies...${TEXT_RESET}"
 
-sudo apt update -y
-sudo apt install -y software-properties-common curl git clang cmake zlib1g-dev zip unzip wget
+if [[ $INSTALL_DEBS != 0 ]]; then
+  sudo apt update -y
+  sudo apt install -y software-properties-common curl git clang cmake zlib1g-dev zip unzip wget
```

Comment:
Should we use the `clang` package for version 14, `clang-14`, to avoid any unexptected errors when `clang` get's updated again, eventually?

Reply:
That might be a good plan, but we should probably wait for a while before doing that since some users might not have Clang 14 in their repos. It'd be nice if we could be tolerant of older versions of LLVM (if that's possible).